libs/date/src/DateInput.tsx

```diff
@@ -318,7 +318,7 @@ function DateInputInput({
     if (showMaskOnEmpty) {
       return undefined;
     }
-    return lang === "en" ? "mm/dd/yyyy" : "dd. mm. yyyy.";
+    return getDateFormatByLang(lang).toLowerCase();
```

Comment:
shouldn't this be `finalDateFormat.toLowerCase()`?

Reply:
I don't have access to that variable anymore since it's defined in a parent component.